jsonschema-core.xml

```diff
@@ -3007,142 +3207,185 @@ https://example.com/schemas/common#/$defs/count/minimum
                     </figure>
                 </section>
 
-                <section title="Detailed">
+                <section title="Hierarchical">
                     <t>
-                        The "Detailed" structure is based on the schema and can be more readable
-                        for both humans and machines.  Having the structure organized this way makes
-                        associations between the errors more apparent.  For example, the fact that
-                        the missing "y" property and the extra "z" property both stem from the same
-                        location in the instance is not immediately obvious in the "Basic" structure.
-                        In a hierarchy, the correlation is more easily identified.
+                        The "Hierarchical" structure is a tree structure that follows the
+                        evaluation path during the validation process.  Typically, it will
+                        resemble the schema as if all referenced schemas were bundled in place
```

Comment:
This paragraph is great, but might I suggest "inlined" instead of "bundled"?  We've been using bundling specifically for the pattern of embedding resources under `$defs`.